src/components/immersive/byline.tsx

```diff
@@ -27,22 +27,22 @@ const Styles = ({ kicker }: PillarStyles): SerializedStyles => css`
 
         time {
             font-size: 1.4rem;
-            color: ${palette.neutral[46]};
+            color: ${neutral[46]};
```

Comment:
You could also use `text.secondary`, which is a bit more semantic

Reply:
👍 I probably should have mentioned in the description that I'm planning to do a second PR and migrate to functional colour names. I was hoping to talk to you first about the dark mode colours that you flagged below.